components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -505,16 +506,26 @@
           _this.sections[name] = {
             expanded: false,
             editable: false,
+            hidden_fields: [],
           };
         } else {
           // If the section is passed in but lacks an "editable" property, assume positive intent.
           if (!Object.keys(_this.sections[name]).includes("editable")) {
             _this.sections[name].editable = true;
           }
+          // Establish a default empty list for hidden fields
+          if (!Object.keys(_this.sections[name]).includes("hidden_fields")) {
+            _this.sections[name].hidden_fields = [];
+          }
         }
       });
     }
   }
+
+  function fieldIsEditable(sectionName: SectionNames, fieldName: string) {
```

Comment:
nit, since we're hiding the field and `sectionName` is just the string key
```suggestion
  function fieldIsHidden(sectionName: string, fieldName: string) {
```

Reply:
@ozsivanov Ah -- I thought that by leaving it as `SectionNames` it would have to conform to one of the actual section names, so if anybody tried to check `fieldIsEditable("foobar")` it would throw a type error